src/MixtureFractionEquationSystem.C

```diff
@@ -387,11 +387,10 @@ MixtureFractionEquationSystem::register_interior_algorithm(
     stk::topology partTopo = part->topology();
     auto& solverAlgMap = solverAlgDriver_->solverAlgorithmMap_;
     
-    AssembleElemSolverAlgorithm* solverAlg = nullptr;
+    AssembleElemSolverAlgorithmNewME* solverAlg = nullptr;
```

Comment:
I think that we need NewME to be the AssembleElemSolverAlg name and deprecate the other...